@here/harp-mapview/lib/Utils.ts

```diff
@@ -410,6 +418,242 @@ export namespace MapViewUtils {
+     *
+     * Here, we fit points into minimal geo box taking world wrapping into account.
+     */
+    export function wrapGeoPointsToScreen(points: GeoCoordLike[]): GeoCoordinates[] {
+        const start = GeoCoordinates.fromObject(points[0]);
+        let north = start.latitude;
+        let south = start.latitude;
+        let lonCenter = MathUtils.normalizeLongitudeDeg(start.longitude);
+        let lonSpan = 0;
+        let east = start.longitude;
+        let west = start.longitude;
+
+        const result: GeoCoordinates[] = [];
+        result.push(new GeoCoordinates(north, lonCenter));
+        for (let i = 1; i < points.length; i++) {
+            const p = GeoCoordinates.fromObject(points[i]);
+            if (p.latitude > north) {
+                north = p.latitude;
+            } else if (p.latitude < south) {
+                south = p.latitude;
+            }
+
+            let longitude = MathUtils.normalizeLongitudeDeg(p.longitude);
+
+            const relToCenter = MathUtils.angleDistanceDeg(lonCenter, longitude);
+            longitude = lonCenter - relToCenter;
+            if (relToCenter < 0 && -relToCenter > lonSpan / 2) {
+                east = Math.max(east, lonCenter - relToCenter);
+                lonSpan = east - west;
+                lonCenter = (east + west) / 2;
+            } else if (relToCenter > 0 && relToCenter > lonSpan / 2) {
+                west = Math.min(west, longitude);
+                lonSpan = east - west;
+                lonCenter = (east + west) / 2;
+            }
+            result.push(new GeoCoordinates(p.latitude, longitude));
+        }
+        return result;
+    }
+
+    /**
+     * Return [[GeoPoints]] bounding [[GeoBox]] applicable for [[getFitBoundsDistance]].
+     *
+     * @returns [[GeoCoordinates]] set that covers `box`
+     */
+    export function geoBoxToGeoPoints(box: GeoBox): GeoCoordinates[] {
+        const center = box.center;
+        return [
+            box.center,
+            new GeoCoordinates(box.north, center.longitude),
+            new GeoCoordinates(box.south, center.longitude),
+            new GeoCoordinates(center.latitude, box.east),
+            new GeoCoordinates(center.latitude, box.west),
+            new GeoCoordinates(box.north, box.west),
+            new GeoCoordinates(box.north, box.east),
+            new GeoCoordinates(box.south, box.west),
+            new GeoCoordinates(box.south, box.east)
+        ];
+    }
+
+    /**
+     * Get minimal distance required for `camera` looking at `worldTarget` to cover `points`.
+     *
+     * All dimensions belong to world space.
+     *
+     * @param points points which shall are to be covered by view
+     *
+     * @param worldTarget readonly, world target of [[MapView]]
+     * @param camera readonly, camera with proper `position` and rotation set
+     * @returns new distance to camera to be used with [[MapView.lookAt]]
+     */
+    export function getFitBoundsDistance(
```

Comment:
Do we need to expose this?
Maybe the two method above are okay, b/c they are quite generic, but this one should be a private method in MapView b/c it is really specific and I don't see any use outside of MapView